package.json

```diff
@@ -47,26 +48,27 @@
         "react-intl": "^6.0.0"
     },
     "devDependencies": {
+        "@babel/helper-builder-react-jsx": "7.15.4",
         "@emotion/react": "^11.8.1",
         "@emotion/styled": "^11.8.1",
         "@mui/icons-material": "^5.5.0",
         "@mui/lab": "^5.0.0-alpha.72",
         "@mui/material": "^5.5.0",
         "@mui/styles": "^5.5.0",
-        "@babel/helper-builder-react-jsx": "7.15.4",
         "@svgr/webpack": "^5.4.0",
         "babel-eslint": "^10.1.0",
         "bufferutil": "^4.0.1",
         "eslint": "^7.9.0",
         "eslint-config-prettier": "^8.0.0",
         "eslint-config-react-app": "^6.0.0",
-        "eslint-webpack-plugin": "^2.0.0",
         "eslint-plugin-flowtype": "^5.2.0",
         "eslint-plugin-import": "^2.22.0",
         "eslint-plugin-jsx-a11y": "^6.3.1",
         "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-react": "^7.21.0",
         "eslint-plugin-react-hooks": "^4.1.2",
+        "eslint-webpack-plugin": "^2.0.0",
+        "jest": "^29.2.2",
```

Comment:
You have too much modifications in package-lock.json, Maybe we should have add jest as a dev dependency in another PR, to separate all this mess from the main scope or this PR.
I don't know if changing dependencies order is a good idea too.

Reply:
I finally removed the tests as it seems not the way.